app.py

```diff
@@ -25,56 +25,64 @@ def str2bool(value: str) -> bool:
     raise ValueError(f"Not a valid boolean: '{value}'")
 
 
-app = core.App()
-
-networking = NetworkingStack(
-    app,
-    "networking",
-    stack_name=f"geospatial-data-lake-networking-{ENV}",
-    env={"region": os.environ["CDK_DEFAULT_REGION"], "account": os.environ["CDK_DEFAULT_ACCOUNT"]},
-    deploy_env=ENV,
-    use_existing_vpc=str2bool(os.environ.get("DATALAKE_USE_EXISTING_VPC", "false")),
-)
-
-storage = StorageStack(
-    app,
-    "storage",
-    stack_name=f"geospatial-data-lake-storage-{ENV}",
-    env={"region": os.environ["CDK_DEFAULT_REGION"], "account": os.environ["CDK_DEFAULT_ACCOUNT"]},
-    deploy_env=ENV,
-)
-
-processing = ProcessingStack(
-    app,
-    "processing",
-    stack_name=f"geospatial-data-lake-processing-{ENV}",
-    env={"region": os.environ["CDK_DEFAULT_REGION"], "account": os.environ["CDK_DEFAULT_ACCOUNT"]},
-    deploy_env=ENV,
-    vpc=networking.datalake_vpc,
-)
-
-api = APIStack(
-    app,
-    "api",
-    stack_name=f"geospatial-data-lake-api-{ENV}",
-    env={"region": os.environ["CDK_DEFAULT_REGION"], "account": os.environ["CDK_DEFAULT_ACCOUNT"]},
-    datasets_table=storage.datasets_table,
-)
-
-api = StagingStack(
-    app,
-    "staging",
-    stack_name=f"geospatial-data-lake-staging-{ENV}",
-    env={"region": os.environ["CDK_DEFAULT_REGION"], "account": os.environ["CDK_DEFAULT_ACCOUNT"]},
-    deploy_env=ENV,
-)
-
-# tag all resources in stack
-core.Tag.add(app, "CostCentre", "100005")
-core.Tag.add(app, "ApplicationName", "geospatial-data-lake")
-core.Tag.add(app, "Owner", "Bill M. Nelson")
-core.Tag.add(app, "EnvironmentType", f"{ENV}")
-core.Tag.add(app, "SupportType", "Dev")
-core.Tag.add(app, "HoursOfOperation", "24x7")
-
-app.synth()
+def main():
```

Comment:
@l0b0 , I think it makes a sense to separate this `app.py` file refactoring to separate PR in case it will cause some issues. Anyway, it is not related to this PR topic.

Reply:
See https://github.com/linz/geospatial-data-lake/pull/202